src/modules/uxrce_dds_client/module.yaml

```diff
@@ -64,3 +64,14 @@ parameters:
             reboot_required: true
             default: 2130706433
             requires_ethernet: true
+
+        UXRCE_DDS_SYNCT:
+            description:
+                short: uXRCE-DDS timesync enable
+                long: |
+                    If set to 1, enable time synchronization between Agent and Client.
```

Comment:
It's a boolean, just explain what happens when it's enabled, I don't think we need to get into 1 vs 0.